README.md

```diff
@@ -1,21 +1,32 @@
-![Aquifer](https://raw.githubusercontent.com/aquifer/aquifer.io/master/branding/aquifer-logo-drupal-1456.png)
+![Aquifer][logo]
```

Comment:
OK, cool, I can get behind that! I like it. 

> Basically I reformatted the readme to be more viewable by terminal editors
I'm not too sure about this, as someone that uses a terminal editor exclusively, I'd rather set my own width and wrapping rules.